portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/retry/ExponentialBackoffStrategy.java

```diff
@@ -14,40 +14,43 @@
  * limitations under the License.
  */
 
-package org.dataportabilityproject.transfer.retry;
+package org.dataportabilityproject.types.transfer.retry;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 
-/**
- * {@link RetryStrategy} that follows an exponential backoff strategy
- */
-public class ExponentialBackoffRetryStrategy implements RetryStrategy {
+public class ExponentialBackoffStrategy implements RetryStrategy {
 
+  @JsonProperty("maxAttempts")
   private int maxAttempts;
+  @JsonProperty("initialIntervalMillis")
   private long initialIntervalMillis;
-  private long multiplier;
+  @JsonProperty("multiplier")
+  private double multiplier;
 
-  public ExponentialBackoffRetryStrategy(int maxAttempts, long initialIntervalMillis,
-      long multiplier) {
+  public ExponentialBackoffStrategy(@JsonProperty("maxAttempts") int maxAttempts,
+      @JsonProperty("initialIntervalMillis") long initialIntervalMillis,
+      @JsonProperty("multiplier") double multiplier) {
     this.maxAttempts = maxAttempts;
     this.initialIntervalMillis = initialIntervalMillis;
     this.multiplier = multiplier;
   }
 
   @Override
   public boolean canTryAgain(int tries) {
+    System.out.println("Tries: " + tries);
```

Comment:
Can we use the logging library and include enough context to make a meaning full message or just dump this line?